assets/source/setup-guide/app/components/Account/Connection.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import apiFetch from '@wordpress/api-fetch';
+import { Spinner } from '@woocommerce/components';
+import { getNewPath } from '@woocommerce/navigation';
+import {
+	Button,
+	CardBody,
+	Flex,
+	FlexItem,
+	FlexBlock,
+	Modal,
+	__experimentalText as Text, // eslint-disable-line @wordpress/no-unsafe-wp-apis --- _experimentalText unlikely to change/disappear and also used by WC Core
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useCreateNotice } from '../../helpers/effects';
+
+const PinterestLogo = () => {
+	return (
+		<>
+			<img
+				src={
+					wcSettings.pinterest_for_woocommerce.pluginUrl +
+					'/assets/images/pinterest-logo.svg'
+				}
+				alt=""
+			/>
+		</>
```

Comment:
Remove unneeded nesting.

```suggestion
			<img
				src={
					wcSettings.pinterest_for_woocommerce.pluginUrl +
					'/assets/images/pinterest-logo.svg'
				}
				alt=""
			/>
```